src/engraving/libmscore/rest.cpp

```diff
@@ -188,7 +188,7 @@ bool Rest::acceptDrop(EditData& data) const
         ) {
         return true;
     }
-    return type != ElementType::SLUR && e->isSpanner();
+    return type != ElementType::SLUR && type != ElementType::TIE && e->isSpanner();
```

Comment:
It seems possible to allow pasting/dropping slurs onto rests, only ties crash. Would we want to allow for that, @wizofaus?

Reply:
I wouldn't change it at this stage, unless you're 100% certain what impacts it will have.
I would definitely write a unit test to ensure we catch cases like this that can cause a crash though!

Reply:
OK, I'm not 100% sure. Only a very basic test showed that it works. And even if it does, it is questionable how often this is reaslly wanted anyway, so let's better not take the risk, not now at least.

Reply:
Feel free to provide a unit test ;-)

Reply:
I'm not in charge of approvals but if I were I'd require one to be submitted along with the fix!  Only reason I hadn't been is that I couldn't work out how to compile/run them previously, but that's working fine now.
But a simple test that creates a tie programmatically and confirms that it's not possible to drop it onto a rest (and cause a crash) shouldn't be hard to write. Key thing is that the test *should* have crashed before this fix.

Reply:
No idea how such a test would need to be made. Also having one after the fix seems pretty pointless?

Reply:
The point of the test is ensure someone else can't make another change in the future that triggers the same behaviour.
This should do it, it crashes for me without your fix:

```
    Element* tie = new Tie(score);
    ChordRest* chordRest = score->firstSegment(SegmentType::ChordRest)->nextChordRest(4);
    ed.dropElement = tie;
    if (chordRest->acceptDrop(ed)) {
        chordRest->drop(ed);
    }
```